### PR TITLE
Rename gpu renderstage producer

### PR DESCRIPTION
**Pull request by rschiu**

### CODE REVIEW

**Comment by lpy** [comment]
Does the latest producer .so drop already have the name changed?

**Comment by rschiu** [comment]
Yes, repackaging it into 9_26 has the name change.  I am repackaging the apk at the moment.

File: core/os/android/adb/commands.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ const (
 	ErrRootFailed      = fault.Const("Device failed to switch to root")
 
 	maxRootAttempts                            = 5
-	gpuRenderingStagesDataSourceDescriptorName = "gpu.renderingstages"
+	gpuRenderingStagesDataSourceDescriptorName = "gpu.renderstages"
```
Comment (lpy) [inline core/os/android/adb/commands.go]: I guess we should change the constant to `gpuRenderStagesDataSourceDescriptorName`

Comment (rschiu) [reply core/os/android/adb/commands.go]: Done

**Comment by pmuetschard** [comment]
What is going on here? is this the final final final name? #3181

**Comment by rschiu** [comment]
Yes, this is the final final final name.  It's always suppose to be this name, but the producer we have didn't use that name until now.

**Comment by lpy** [comment]
The reason I changed it to `renderingstages` before was because our own prototype https://github.com/rschiu/perfetto-sdk/blob/master/android_client_api_example.cc#L104 also uses this name :D sorry for the confusion

**Pull request merged**